src/app/components/speeddial/speeddial.ts

```diff
@@ -419,7 +421,8 @@ export class SpeedDial implements AfterViewInit, AfterContentInit, OnDestroy {
 
     onBlur(event) {
         this.focused = false;
-        this.focusedOptionIndex.set(-1);
+        // fix error: NG0600: Writing to signals is not allowed in a computed or an effect by default
```

Comment:
Can you please remove this comment, it's not needed

Reply:
Absolutely.